src/resolve/resolved-manager.c

```diff
@@ -1054,6 +1054,60 @@ static int manager_ipv6_send(
         return sendmsg_loop(fd, &mh, 0);
 }
 
+int send_dns_notification(Manager *m, DnsAnswer *answer, const char *query_name) {
+        _cleanup_free_ char *normalized = NULL;
+        DnsResourceRecord *rr;
+        int ifindex, r;
+        _cleanup_(json_variant_unrefp) JsonVariant *array = NULL;
+        Varlink *connection = NULL;
+
+        assert(m);
+
+        DNS_ANSWER_FOREACH_IFINDEX(rr, ifindex, answer) {
+                _cleanup_(json_variant_unrefp) JsonVariant *entry = NULL;
+                int family;
+
+                if (rr->key->type == DNS_TYPE_A) {
+                        struct in_addr *addr = &rr->a.in_addr;
+                        family = AF_INET;
```

Comment:
It is not necessary to declare `family`, why not directly pass `AF_INET` to `json_build()`?